### ISSUE TITLE
Stuck on Building Image Page

### ISSUE THREAD

**Issue by victoriajames3** [gitpod-io/gitpod#5020]
### Bug description

I cannot open any workspaces. It has been stuck on building image since 2pm yesterday. I cannot delete my workspaces either to try and open new ones. The delete workspace button is simply inactive

### Steps to reproduce

Try to open a workspace. 

### Expected behavior

I should be able to open my workspace in Gitpod

**Comment by Sharon-B** [comment]
I am also having the same issue since yesterday afternoon. Workspace stuck on building image for over 3 hours. Will there be any way to access this workspace again?

**Comment by meysholdt** [comment]
Hi @Sharon-B , hi @victoriajames3 !

There was a disk space Issue related to the image builder. I've cleaned the disks. Can you try again?

Apologies for the trouble.

**Comment by Sharon-B** [comment]
Hi @meysholdt,

I still have the same issue with that workspace, it's stuck on building image when I try to open it.

Thanks for your assistance on this.

**Comment by victoriajames3** [comment]
Hi @meysholdt 
Thanks for getting back to me 
I can create a new workspace now however I still cannot open the existing ones. However within the new workspace, Emmet isn't working and neither is the preview. I am using the python3 -m http.server command to create the preview window however I cannot preview any files I create. Sorry I am quite new at this but I have tried reloading, creating a new workspaces. Also, as soon as I leave gitpod. Those new workspaces I created just disappear and I am left with the ones I can't open. 

Thanks for your help with this
